### PR TITLE
withLogger

### LINKED ISSUE #18: Rename named export to withLogger.
### Summary

Let's rename the named export to `withLogger` and this way, start a naming convention for all future higher-order-apps, e.g. `window.withGizmo`. I am not proposing we add a prefix to every package; I think `@hyperapp/html` should continue to be available as `window.html`.

### Related

- https://github.com/HyperappCommunity/hyperapp-debug/issues/1

### PR DESCRIPTION
**Pull request by okwolf**
Refactor default export to `withLogger`.
Also `window.logger` to `window.hyperappLogger`.

Closes #18.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,42 +1 @@
...
-      }
-
-      var enhancedActions = enhanceActions(actionsTemplate)
-
-      var appActions = app(initialState, enhancedActions, view, container)
-      return appActions
-    }
-  }
-}
+export { withLogger } from "./withLogger"
```
Comment (frenzzy) [inline src/index.js]: Why do you need this re-export file, why not just include logger implementation here or remove this file at all?
It is so annoying when you click on import in IDE and you see such file when you expect to see implementation.

Comment (jorgebucaran) [reply src/index.js]: Agreed. 💯

Comment (okwolf) [reply src/index.js]: Guess I've got a bad habit of always using `index.js` for my exports to the outside world and naming other files by what they export 😬 

Changed.

**Review by frenzzy — APPROVED** [review]

File: src/index.js (hunk 1 above)
Comment (jorgebucaran) [reply src/index.js]: @okwolf What if you rename it to `index.js`, but export `withLogger` from it? Like how we do in hyperapp itself. Would that be better? I think so.

Comment (okwolf) [reply src/index.js]: @jorgebucaran take a look at my newest commit, I think you'll approve.

**Comment by okwolf** [comment]
@jorgebucaran any issues with merging this soon? Likely when I'm more awake.

😴 🐺

File: README.md [hunk 2]
```diff
@@ -35,12 +63,23 @@ Download the minified library from the [CDN](https://unpkg.com/@hyperapp/logger)
...
 
 ## Usage
 
+Compose the `withLogger` function with your `app` before calling it with the usual arguments.
+
 ```js
-logger(options)(app)(state, actions, view, document.body)
+import { withLogger } from "@hyperapp/logger"
+
+withLogger(options)(app)(state, actions, view, document.body)
```
Comment (frenzzy) [inline README.md]: Maybe make focus on simplicity - show "no options" example first and later "if you need options you can use this signature"

Comment (okwolf) [reply README.md]: I switched the order now, what do you think?

**Pull request merged**